Antidote/CallsManager.m

```diff
@@ -59,14 +59,18 @@ - (instancetype)init
                                                                delegate:self];
 
     NSPredicate *predicate = [NSPredicate predicateWithFormat:@"status == %d", OCTCallStatusActive];
+    NSArray *sortDescriptors = @[
+        [RLMSortDescriptor sortDescriptorWithProperty:@"pausedStatus" ascending:YES]
```

Comment:
The reason I did that is because if we sort it, we can prioritize which call is queued up next to show on the screen in the below method.

`- (void)didRemoveCurrentCall
{
    OCTCall *call = [self.allActiveCallsController.fetchedObjects firstObject] ?:
                    [self.allCallsController.fetchedObjects firstObject];
...`

For example in the above, since it's sorted, it will pull the first non-paused call from the `allActiveCallsController` list.